npm/packages/@socketsupply/socket-node/index.js

```diff
@@ -144,13 +167,13 @@ class API {
 
     if (s.length > MAX_MESSAGE_KB) {
       const len = Math.ceil(s.length / 1024)
-      process.stderr.write('WARNING: Sending large message to webview: ' + len + 'kb\n')
-      process.stderr.write('RAW MESSAGE: ' + s.slice(0, 512) + '...\n')
+      this.#writeStderr(
```

Comment:
@chicoxyzzy same here

Reply:
Yes, we can drop both. I don't know why we have such limit.

Reply:
@heapwolf maybe you know why?

Reply:
It was left over from the legacy module. We can drop it. @aleclarson could you remove the limit and warning?

Reply:
Let’s leave it for now. I will check if it’s necessary with #666

Reply:
SGTM